lib/client/index.js

```diff
@@ -5,7 +5,7 @@ var _ = require('lodash')
   , defaultConfig = require('../config')
   , Rx = require('rx')
   , DataManager = require('./data-manager');
-
+var debug = require('../utils/logger')(__filename);
```

Comment:
it passes it to the logger signature generator, such that we don't need constants etc.